tensorflow/python/estimator/inputs/pandas_io.py

```diff
@@ -117,7 +138,12 @@ def input_fn():
     features = features[1:]
     features = dict(zip(list(x.columns), features))
     if y is not None:
-      target = features.pop(target_column)
+      if isinstance(target_column, list):
```

Comment:
in order to make this check valid, it is critical to add a precheck above to ensure the target_column passed in must not be a list.

Reply:
Should it be valid for the `target_column` passed to be a list? I don't think it was valid before so this wouldn't be a behavior change so adding the precheck makes sense. Will do unless there's some compelling reason it should be able to be a list.